packages/ui-avatar/src/components/MessageAvatar.tsx

```diff
@@ -0,0 +1,31 @@
+import { AvatarContainer, Box } from '@rocket.chat/fuselage';
+import type { ReactElement, VFC } from 'react';
+
+import { UiAvatarProps } from './Avatar';
+import UserAvatar from './UserAvatar';
+
+type MessageAvatarProps = {
+	emoji?: ReactElement;
+	avatarUrl?: string;
+	username: string;
+	onClick?: (e: any) => void;
+	size?: UiAvatarProps['size'];
+};
+
+const MessageAvatar: VFC<MessageAvatarProps> = ({ emoji, avatarUrl, username, size = 'x36', onClick, ...props }) => {
+	if (emoji) {
+		return (
+			<AvatarContainer size={size} {...props}>
+				{emoji}
+			</AvatarContainer>
+		);
+	}
+
+	return (
```

Comment:
we could avoid this `Box`

Reply:
The only component who need a onClick event is the MessageAvatar, why should I pass down to the base Avatar the unnecessary event?